package.json

```diff
@@ -1,5 +1,5 @@
 {
   "name": "@chainlink/solhint-plugin-chainlink-solidity",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
Manually bumped this as although there is an automated Changesets check, it doesn't look like Changesets are actually supported by this repo